packages/liferay-js-publish/src/index.js

```diff
@@ -175,11 +171,53 @@ function getRemote() {
 	throw new Error('Unable to determine remote repository URL');
 }
 
+function isPrereleaseVersion(pkg) {
+	const {version} = pkg;
+
+	const parts = version.split('.');
+
+	// De facto standard for prerelease versions is; 10.0.0-alpha.1
+
+	if (parts.length > 3) {
+		return true;
+	}
+
+	// look for any non-number character
```

Comment:
Comment doesn't match what the code does. You're looking for a hyphen.

Reply:
Mmm, I changed the code in the last moment :sweat_smile: